api/v1alpha1/validation.go

```diff
@@ -485,3 +522,8 @@ func getAppName(app ApplicationSpec, appType ApplicationProviderType) (string, e
 		return "", fmt.Errorf("unsupported application provider type: %s", appType)
 	}
 }
+
+func validateTimezone(tz string) error {
+	_, err := time.LoadLocation(tz)
+	return err
```

Comment:
I'm not quite familiar with 'LoadLocation' errors; we might consider wrapping them.

if _, err := time.LoadLocation(tz); err != nil {
		return fmt.Errorf("invalid timezone: %s", tz)
	}
	return nil